exercises/practice/difference-of-squares/.meta/example.cairo

```diff
@@ -0,0 +1,35 @@
+pub fn squareOfSum(number: u64) -> u64 {
+    let mut sum = 0;
+    let mut current = 1;
+
+    loop {
+        if current > number {
+            break;
+        }
+
+        sum += current;
+        current += 1;
+    };
+
+    return sum * sum;
+}
+
+pub fn sumOfSquares(number: u64) -> u64 {
```

Comment:
There's actually a formula for this too, it's called a "pyramid number".

Using this the function becomes a 1-liner:
```rust
// See https://en.wikipedia.org/wiki/Square_pyramidal_number
pub fn sumOfSquares(number: u64) -> u64 {
    number * (number + 1) * (2 * number + 1) / 6
}
```